staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4996,11 +4996,21 @@ type EndpointPort struct {
 	// +optional
 	Protocol Protocol `json:"protocol,omitempty" protobuf:"bytes,3,opt,name=protocol,casttype=Protocol"`
 
-	// The application protocol for this port.
```

Comment:
I think I'd keep this sentence.  e.g. 

"""
The application protocol for this port.  This is used as a hint for implementations to offer richer behavior for protocols that they understand.
"""